### ISSUE TITLE
Hello app does not work

### ISSUE THREAD

**Issue by BillShiyaoZhang** [Chainlit/chainlit#1543]
**Describe the bug**
The hello app does not work.
```
>> $ chainlit hello
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.13/bin/chainlit", line 5, in <module>
    from chainlit.cli import cli
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/chainlit/__init__.py", line 18, in <module>
    from chainlit.action import Action
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/chainlit/action.py", line 5, in <module>
    from chainlit.telemetry import trace_event
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/chainlit/telemetry.py", line 12, in <module>
    from chainlit.config import config
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/chainlit/config.py", line 538, in <module>
    config = load_config()
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/chainlit/config.py", line 507, in load_config
    settings = load_settings()
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/chainlit/config.py", line 479, in load_settings
    code_settings = CodeSettings(action_callbacks={})
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pydantic/_internal/_dataclasses.py", line 121, in __init__
    s.__pydantic_validator__.validate_python(ArgsKwargs(args, kwargs), self_instance=s)
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pydantic/_internal/_mock_val_ser.py", line 100, in __getattr__
    raise PydanticUserError(self._error_message, code=self._code)
pydantic.errors.PydanticUserError: `CodeSettings` is not fully defined; you should define `Action`, then call `pydantic.dataclasses.rebuild_dataclass(CodeSettings)`.

For further information visit https://errors.pydantic.dev/2.10/u/class-not-fully-defined
```

**To Reproduce**

1. pip install chainlit
2. chainlit hello
3. See error

**Expected behavior**
The hello app successfully runs in browser

**Desktop:**
- OS: MacOS Sequoia v15.1.1
- Browser Safari v18.1.1
- Chip: M1

**Comment by Incospro-Jan** [comment]
Hi @BillShiyaoZhang ,

I just had the same error log on an existing Chainlit based application.
Starting about 3 hours ago when pushing updates and re-deploying the application was crashing with the exact same error.

I looked at the CI/CD logs and found that the **pydantic** version changed from `2.10.1` to `2.10.2`.
After pinning the version in the requirements.txt to `2.10.1` I was able to succesfully re-deploy.
`pydantic==2.10.1`
Maybe give that a try and let me know if the error disappears.
